From 45389cea6b165ba53b2417f75eabef90dfc12d96 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Mon, 13 Apr 2020 11:11:25 +0800
Subject: [PATCH 2/2] can: rx-offload: drop unused variable 'skb_error'

In order to fix below warning:
drivers/net/can/rx-offload.c:134:31: warning: unused variable 'skb_error' [-Wunused-variable]
  134 |  struct sk_buff *skb = NULL, *skb_error = NULL;
      |                               ^~~~~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/can/rx-offload.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/can/rx-offload.c b/drivers/net/can/rx-offload.c
index c9ed049902b4..ab7a24233c74 100644
--- a/drivers/net/can/rx-offload.c
+++ b/drivers/net/can/rx-offload.c
@@ -131,7 +131,7 @@ static int can_rx_offload_compare(struct sk_buff *a, struct sk_buff *b)
 static struct sk_buff *
 can_rx_offload_offload_one(struct can_rx_offload *offload, unsigned int n)
 {
-	struct sk_buff *skb = NULL, *skb_error = NULL;
+	struct sk_buff *skb = NULL;
 	u32 timestamp;
 	bool drop;
 
-- 
2.17.1

